netutils/config/compliance.py

```diff
@@ -172,14 +180,16 @@ def compliance(features, backup, intended, network_os, cfg_type="file"):
     compliance_results = {}
 
     for feature in features:
-        backup_str = section_config(feature, backup_cfg, network_os)
-        intended_str = section_config(feature, intended_cfg, network_os)
+        backup_str = section_config(feature, backup_cfg, network_os)  # type: ignore
+        intended_str = section_config(feature, intended_cfg, network_os)  # type: ignore
         compliance_results.update({feature["name"]: feature_compliance(feature, backup_str, intended_str, network_os)})
 
-    return compliance_results
+    return compliance_results  # type: ignore
 
 
-def config_section_not_parsed(features, device_cfg, network_os):
+def config_section_not_parsed(
+    features: t.List[t.Dict[str, t.Union[str, bool, t.List[str]]]], device_cfg: str, network_os: str
```

Comment:
Do we want `network_os` to be an Enum type? This way we further restrict possible values for it and make sure we don't accidentally pass wrong types to it. Also having `netutils` possibly even export such an Enum for use within importing libraries makes conceptual sense to me.